src/widgets/stadiumlistitem.hpp

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <QTreeWidgetItem>
+#include "src/datastore/stadium.hpp"
+#include "src/datastore/team.hpp"
+
+class StadiumListItem : public QTreeWidgetItem
+{
+public:
+    /* Constructor */
+    StadiumListItem(QTreeWidget* parent, const Stadium&, const QStringList&);
+
+    /* Operator Overloads*/
+    virtual bool operator<(const StadiumListItem&);
```

Comment:
do not mark `virtual`, mark `override` instead

Reply:
needs to be virtual for calls, but ancestor not virtual (so cannot use overload specifier https://en.cppreference.com/w/cpp/language/override)

Reply:
Oh didn't notice this isn't overriding the base class one, ok then